test/e2e/dns_ingressdegrade_test.go

```diff
@@ -39,6 +40,23 @@ func TestIngressStatus(t *testing.T) {
 		t.Fatalf("failed to get DNS config: %v", err)
 	}
 
```

Comment:
You should be able to remove the `Get(..., ..., &dnsConfig)` above since `TestMain` initializes `dnsConfig`, but we can keep it if you prefer to keep the test logic self-contained.  

For that matter, we should use `kclient` that `TestMain` also initializes, but that can be left as a follow-up clean-up.

Reply:
I'll try doing it here

Reply:
I'll follow-up with a PR to address both comments. That'll help me to iterate at the same time on my other PR for LB.
Thanks